### PR TITLE
Rewriting code

### PR DESCRIPTION
**Pull request by DimaKacer**
- create folder header and files for module header
- create folder footer and files for module footer

### CODE REVIEW

File: angular-spa-ui/img/index.html [hunk 1]
```diff
@@ -1 +1 @@
-"use strict";
+<!DOCTYPE html><title></title>
```
Comment (tarhunakau) [inline angular-spa-ui/img/index.html]: No empty line at the end of file

File: angular-spa-ui/js/common/footer/footer-ctrl.js [hunk 2]
```diff
@@ -0,0 +1 @@
+"use strict";
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/footer/footer-ctrl.js]: No empty line at the end of file

File: angular-spa-ui/js/common/footer/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+"use strict";
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/footer/index.js]: No empty line at the end of file

File: angular-spa-ui/js/common/header/header-ctrl.js [hunk 4]
```diff
@@ -0,0 +1 @@
+"use strict";
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/header/header-ctrl.js]: No empty line at the end of file

File: angular-spa-ui/js/common/header/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+"use strict";
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/header/index.js]: No empty line at the end of file

File: angular-spa-ui/js/search-app/cart-module/cart.jade [hunk 6]
```diff
@@ -1 +1 @@
-<p>{{greeting}}</p>
\ No newline at end of file
+<h2>{{greeting}}</h2>
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/cart-module/cart.jade]: No empty line at the end of file

File: angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade [hunk 7]
```diff
@@ -1 +1 @@
-<p>{{greeting}}</p>
\ No newline at end of file
+<h2>{{greeting}}</h2>
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade]: No empty line at the end of file

File: angular-spa-ui/js/search-app/search-module/search.jade [hunk 8]
```diff
@@ -1 +1 @@
-p {{greeting}}
\ No newline at end of file
+<h2>{{greeting}}</h2>
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/search.jade]: No empty line at the end of file

File: angular-spa-ui/img/index.html (hunk 1 above)
Comment (tarhunakau) [inline angular-spa-ui/img/index.html]: Wrap <title> in <head> tag, add meta

File: angular-spa-ui/js/search-app/cart-module/cart.jade (hunk 6 above)
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/cart-module/cart.jade]: Variable in brackets should be separated with space, e.g. <h2>{{ greeting }}</h2>

File: angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search-ctrl.js [hunk 9]
```diff
@@ -1 +1,6 @@
 "use strict";
+
+module.exports = function ($scope) {
+    console.log( $scope );
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search-ctrl.js]: You do not need to separate function argument with spaces, it should looks: console.log($scope) and double check do you really need this debug information in pull request.

File: angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade (hunk 7 above)
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade]: Variable in brackets should be separated with space, e.g. <h2>{{ greeting }}</h2>

File: angular-spa-ui/js/search-app/search-module/search.jade (hunk 8 above)
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/search.jade]: Variable in brackets should be separated with space, e.g. <h2>{{ greeting }}</h2>

File: angular-spa-ui/js/search-app/tags-module/tags.jade [hunk 10]
```diff
@@ -1 +1 @@
-<p>{{greeting}}</p>
\ No newline at end of file
+<h2>{{greeting}}</h2>
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/tags-module/tags.jade]: Variable in brackets should be separated with space, e.g. <h2>{{ greeting }}</h2>

**Comment by tarhunakau** [comment]
Fix all remarks and pay attention: a lot of commits with a lot of merges is not applicable for any projects. You do not need to revert it and make with 1-2 merges at the moment, that is information for future.

File: angular-spa-ui/img/index.html (hunk 1 above)
Comment (ValeryShvyndzikau) [inline angular-spa-ui/img/index.html]: Why index.html file placed into the  /img folder?

File: angular-spa-ui/js/common/footer/footer-ctrl.js (hunk 2 above)
Comment (ValeryShvyndzikau) [inline angular-spa-ui/js/common/footer/footer-ctrl.js]: Please rename pull request properly, "rewrite cod" is contains typos

File: angular-spa-ui/js/search-app/search-config.js [hunk 11]
```diff
@@ -1 +1,8 @@
 "use strict";
+
+module.exports = function () {
```
Comment (ValeryShvyndzikau) [inline angular-spa-ui/js/search-app/search-config.js:3]: Search config should export object, not a function

File: angular-spa-ui/img/index.html (hunk 1 above)
Comment (DimaKacer) [inline angular-spa-ui/img/index.html]: Fixed.
- removed index.html;
- added file .png

**Comment by ValeryShvyndzikau** [comment]
Each title of commit or pull request should starts by Capital letter.

File: angular-spa-ui/js/common/footer/index.js (hunk 3 above)
Comment (DimaKacer) [inline angular-spa-ui/js/common/footer/index.js]: fixed

File: angular-spa-ui/js/common/header/header-ctrl.js (hunk 4 above)
Comment (DimaKacer) [inline angular-spa-ui/js/common/header/header-ctrl.js]: fixed

File: angular-spa-ui/js/common/header/index.js (hunk 5 above)
Comment (DimaKacer) [inline angular-spa-ui/js/common/header/index.js]: fixed

File: angular-spa-ui/js/search-app/cart-module/cart.jade (hunk 6 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/cart-module/cart.jade]: fixed

File: angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade (hunk 7 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search.jade]: fixed

File: angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search-ctrl.js (hunk 9 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search-ctrl.js]: fixed

File: angular-spa-ui/js/search-app/search-module/search.jade (hunk 8 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/search-module/search.jade]: fixed

File: angular-spa-ui/js/search-app/tags-module/tags.jade (hunk 10 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/tags-module/tags.jade]: fixed

File: angular-spa-ui/js/search-app/search-config.js (hunk 11 above)
Comment (DimaKacer) [inline angular-spa-ui/js/search-app/search-config.js:3]: +

File: angular-spa-ui/js/common/footer/footer-ctrl.js (hunk 2 above)
Comment (DimaKacer) [inline angular-spa-ui/js/common/footer/footer-ctrl.js]: +

**Comment by DimaKacer** [comment]
Rewriting code

**Pull request merged**